client/src/css/structure.css

```diff
@@ -185,6 +185,8 @@
   border: none;
   max-height : 42px;
   border-bottom : 1px solid #ccc;
+  min-height: 35px;
+  padding-top: 7px;
```

Comment:
Maybe the `flex-util` need some fixation, the `padding-top: 7px` is necessary to avoid to have something like this : 

![padding](https://cloud.githubusercontent.com/assets/5445251/16707842/d8892cf8-4591-11e6-9e78-503b63d27d6d.png)

The content of the util bar is so closed to the top of the `flex-util` bar, the `padding-top: 7px;` is not so good, maybe i can use `line-height: 35px` to center the content in the util bar.

All proposal are welcome